frontend/tests/controllers/UserRegistrationTest.php

```diff
@@ -186,4 +186,33 @@ public function testUserParentalTokenNotGeneratedDueInvalidParameters() {
             $this->assertSame('parameterInvalid', $e->getMessage());
         }
     }
+
+    /**
+     * User 13 and its account link to the parent account
+     * on verification of parental token
+     */
+    public function testUse13LinkedToParentAccountWhenTokenVerificationDone() {
+        $defaultDate = strtotime('2022-01-01T00:00:00Z');
+        \OmegaUp\Time::setTimeForTesting($defaultDate);
+        // Create a 10 years-old user
+        ['identity' => $identity] = \OmegaUp\Test\Factories\User::createUser(
+            new \OmegaUp\Test\Factories\UserParams([
+                'birthDate' => strtotime('2012-01-01T00:00:00Z'),
+            ]),
+        );
+        $login = self::login($identity);
+
+        $payload = \OmegaUp\Controllers\User::getVerificationParentalTokenDetailsForTypeScript(
+            new \OmegaUp\Request([
+                'auth_token' => $login->auth_token,
+            ])
+        )['templateProperties']['payload'];
+        //verify token belongs to child accounts ---?
```

Comment:
This function should receive the token, so you need to add it in the request:
```suggestion
        $payload = \OmegaUp\Controllers\User::getVerificationParentalTokenDetailsForTypeScript(
            new \OmegaUp\Request([
                'auth_token' => $login->auth_token,
                'parental_verification_token' => $user->parental_verification_token,
            ])
        )['templateProperties']['payload'];

```

Reply:
oh yess! I forgot to add. got that!